packages/components/src/ObjectViewer/JSONLike/JSONLike.component.js

```diff
@@ -255,7 +249,7 @@ export function ComplexItem({ data, name, opened, edited, jsonpath, info, onSele
 						label={getDataAbstract(data)}
 						tooltipPlacement="right"
 					>
-						<sup className={badgeClasses}>{decoratedLength}</sup>
+						<sup className="badge">{decoratedLength}</sup>
```

Comment:
shouldn't it be theme['badge']?

Reply:
there is no 'badge' class in the theme file, this is a bootstrap class.